### PR TITLE
Disallow unitless slices. Only allow px.

### PR DESCRIPTION
**Pull request by betaboon**
Implements solution discussed in #30

### CODE REVIEW

File: index.scss [hunk 1]
```diff
@@ -198,14 +199,29 @@ $media: null !default;
...
+/// @access private
+/// @arg {Any} $value - a value to check
+/// @return {Bool} whether the value is a number with px unit
+/// @example sass
+///   is-number-px(100px) // => true
+///   is-number-px(100em) // => false
+///   is-number-px(100) // => false
+///   is-number-px('text') // => false
+@function is-number-px($value) {
+  @return meta.type-of($value) == 'number' and unit($value) == 'px';
```
Comment (lolmaus) [inline index.scss]: You're using two functions here, one namespaced and one not.

Ideally, all function calls should be namespaced for clarity.

**Review by lolmaus — CHANGES_REQUESTED** [review]
Great work, thank you so much!

I've got one request in line comments.

File: index.scss (hunk 1 above)
Comment (betaboon) [reply index.scss]: oh that must have slipped my mind after i got distracted looking at `map-sort-values` where none of the `list` functions are namespaced as opposed to everywhere else.

i will fix that here.
should i add another commit that namespaces the function calls in `map-sort-values` ?

Comment (lolmaus) [reply index.scss]: > should i add another commit that namespaces the function calls in map-sort-values ?

Yes please! 🙇

Comment (betaboon) [reply index.scss]: done and done.

i thought you did that on purpose as not to change the code that you attributed to be taken from somewhere else :)

Comment (lolmaus) [reply index.scss]: I copied it over without realizing the difference in code style.

But I think I'd rather keep the style consistent. There is no benefit in preserving original code unchanged. 🤷 

Thanks again!

**Review by lolmaus — APPROVED** [review]

**Comment by betaboon** [comment]
just a reminder to mention this in the changelog as breaking change :)

**Comment by lolmaus** [comment]
> just a reminder to mention this in the changelog as breaking change :)

Done! Released as `3.0.0-beta.1`.

**Comment by betaboon** [comment]
thank you very much. It's been a pleasure :1st_place_medal:

**Pull request merged**